spec/rspec/core/memoized_helpers_spec.rb

```diff
@@ -363,6 +363,97 @@ def not_ok?; false; end
         expect(subject).to eq(3)
       end
     end
+
+    describe 'threadsafety', :threadsafe => true do
+      # Force call to eq to happen in threaded env to prevent "circular require" warnings
+      # Not actually sure why calling eq in two threads at once causes this (it's a sporadic failure):
```

Comment:
My default is still 2.1 (only b/c I haven't updated it), so it's possible that I was using that when it failed. But since we test as far back as 1.8.7, we would need to deal with it either way.